### PR TITLE
BACnet Binding

### LINKED ISSUE #144: BACnet Binding for Web of Things
I like to start the initiative to formalize an official BACnet binding for the Web of Things. BACnet is an important protocol for building automation and control systems for scenarios like heating, ventilating, and air-conditioning control (HVAC), lighting control, and fire detection systems. Standardized BACnet-based metadata for WoT Thing Descriptions would simplify the onboarding and use of BACnet-based systems for various IoT applications (also see [Smart Buidling](https://w3c.github.io/wot-usecases/#smart-buildings) and [Protocol-Interworking](https://w3c.github.io/wot-usecases/#X-Protocol-Interworking) use cases).  

This thread should be used to collect material, comments, and people who are interested in this initiative.

### PR DESCRIPTION
**Pull request by ektrah**
Work in progress

Resolves #144

### CODE REVIEW

File: bindings/protocols/bacnet/index.html [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+                w3cid: "107859",
+                company: "Siemens AG",
+                companyURL: "https://www.siemens.com/"
+            },
+            {
+                name: "Michael Koster",
+                w3cid: "110658",
+                company: "Invited Expert",
+                companyURL: "https://github.com/mjkoster"
+            }],
```
Comment (fennibay) [inline bindings/protocols/bacnet/index.html:22]: Now I have a W3C id:

```suggestion
            },
            {
                name: "Dogan Fennibay",
                w3cid: "147431",
                company: "Siemens AG",
                companyURL: "https://www.siemens.com/"
            }],
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /bindings/payloads/xml/ @egekorkan @mjkoster
 /bindings/platforms/hue/ @egekorkan @mjkoster
+/bindings/protocols/bacnet/ @ektrah @mjkoster
```
Comment (fennibay) [inline .github/CODEOWNERS:3]:
```suggestion
/bindings/protocols/bacnet/ @ektrah @mjkoster @fennibay 
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /bindings/payloads/xml/ @egekorkan @mjkoster
 /bindings/platforms/hue/ @egekorkan @mjkoster
-/bindings/protocols/bacnet/ @ektrah @mjkoster
+/bindings/protocols/bacnet/ @ektrah @mjkoster @fennibay
```
Comment (danielpeintner) [inline .github/CODEOWNERS:3]: @fennibay I think _someone_ needs no add you as contributor to this GitHub repo

Comment (egekorkan) [reply .github/CODEOWNERS:3]: The proposal is to wait to merge this PR, at that point @fennibay will be an editor, thus making sense to give him write access to the repo.

**Comment by egekorkan** [comment]
Call of 04.10:
- The group is fine to merge but we need to solve two issues (that will be submitted by @egekorkan).

**Pull request merged**